input/fsh/au-erequesting-path-request.fsh

```diff
@@ -0,0 +1,23 @@
+Profile: AUeRequestingPathologyServiceRequest
+Parent: AUeRequestingServiceRequest
+Id: au-erequesting-path-request
+Title: "AU eRequesting Pathology ServiceRequest"
+Description: "This profile sets minimum expectations for a ServiceRequest resource that is used to record, search, and fetch information about pathology test requests, plans, or orders for a service for a patient. It is based on the [AU eRequesting ServiceRequest](https://build.fhir.org/ig/hl7au/au-fhir-erequesting/StructureDefinition-au-erequesting-servicerequest.html) and identifies the additional constraints, extensions, vocabularies and value sets that **SHALL** be present in the ServiceRequest when conforming to this profile."
+
+* ^status = #draft
+
+* ^extension[http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm].valueInteger = 0
+
+* . ^short = "A pathology test request"
+* . ^definition = "An order or referral for a pathology or laboratory service."
```

Comment:
The definition should say "A request for a pathology service."

Reply:
Done